### PR TITLE
Adding VERSION unique variable to set all the charts version.

### PR DESCRIPTION
**Pull request by Andrea-Campanella**
Enable to deploy a specific set of charts without modifying all of those variables.

### CODE REVIEW

File: voltha [hunk 1]
```diff
@@ -64,6 +64,14 @@ if [ $FANCY -eq 1 ]; then
     CIVIS=$(tput civis)
 fi
 
+if [ ! -z "$VERSION" ]; then
```
Comment (teone) [inline voltha]: `VERSION` needs to be added to the `ALL_OPTIONS` list so that it's exported in `$TYPE-env.sh`

File: voltha [hunk 2]
```diff
@@ -64,6 +64,14 @@ if [ $FANCY -eq 1 ]; then
...
+    VOLTHA_CHART_VERSION=$VERSION
+    VOLTHA_BBSIM_CHART=$VERSION
```
Comment (teone) [inline voltha]: BBSim chart is `3.0.7`

File: voltha [hunk 3]
```diff
@@ -64,6 +64,14 @@ if [ $FANCY -eq 1 ]; then
...
+    VOLTHA_ADAPTER_SIM_CHART_VERSION=$VERSION
```
Comment (teone) [inline voltha]: `2.2.3`

**Comment by Andrea-Campanella** [comment]
@davidkbainbridge this is ready for merging if you want. Thanks

**Pull request merged**